src/components/dialog.js

```diff
@@ -73,10 +73,13 @@ const resetViewStyles = {
  * - desktopConfig: Options for dialogs on desktop screens.
  * - maxAllowedHeightRatio: The max allowed height of the view as a ratio of the
  *       viewport height.
+ * - iframeCssClass: The CSS class for the iFrame. Defaults to 'swg-dialog' if
```

Comment:
hmm, I wonder if iframeCssClassOverride is a better name for this property, because it overrides both 'swg-dialog' and the 'swg-dialog swg-wide-dialog' combo. in the future, there may be other configuration options that add CSS classes, too, and setting this property will override all of them.

Reply:
Good point, what if this param just added additional classes instead of overriding the default ones?

Reply:
Good point! I renamed to iframeCssClassOverride as for my use case, the styles provided by swg-dialog are not needed, so I wanted to be able to override with a different class.